NETPROGRAM/String_Exercise/Exercise04.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NETPROGRAM.String_Exercise
+{
+    public class Exercise04
+    {
+
+        public int PowInt(int n,int m)
+        {
+            var mul = 1;
+                      
+            for(var i = 1; i <= m; i++)
+            {
+                mul *= n;
+            }
+
+            return mul;
+        }
+
+        public int SumIntInString(string s)
+        {
+            var sum = 0;
+            var lengInt = 0;
+
+            foreach(var ch in s.ToCharArray().Reverse())
```

Comment:
Giải pháp này không tốt, thử tìm cách khác, tính toán từ trái sang phải thay vì reverse

Reply:
done. Dùng cách mới anh + thêm một ký tự cuối dòng, để duyệt code đẹp hơn. Nếu không nên sử dụng như vậy thì anh edit lại và thêm điều kiện vào vòng for trong.